recipes/hpv-kite/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "hpv-kite" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hpdcj/HPV-KITE/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e928ceb741b4b1fd3ade33e6204530e5998c4b5091f4a9d7f2e094bdef897ec7
+
+build:
```

Comment:
```suggestion
build:
  noarch: generic
```

Reply:
This should be `noarch: generic`, and you can then remove the `bld.bat`.  The Windows failure can be ignored in that case, as the build will only be run on `linux-64` in the actual feedstock.